templates/pam_su.erb

```diff
@@ -0,0 +1,68 @@
+#
```

Comment:
This template is only valid for debian / ubuntu ... please also distinguish between the distributions. I'll attach the default settings for SUSE and RedHat/Centos.